src/renderer/src/routes/pages/Wallets/index.tsx

```diff
@@ -52,7 +53,33 @@ export const WalletsPage = () => {
       }
       rightComponent={
         <div className="flex gap-x-2">
-          <IconButton icon={<TbMenuDeep />} filled={false} size="md" text={t('manageButtonLabel')} disabled />
+          <PopOver
+            trigger={
+              <IconButton
+                icon={<TbMenuDeep />}
+                filled={false}
+                size="md"
+                text={t('manageButtonLabel')}
+                className="data-[state=open]:bg-gray-200/15"
```

Comment:
It is not necessary since you are passing the activated as prop to trigger inside PopOver